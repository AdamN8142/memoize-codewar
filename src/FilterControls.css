.controls-container {
  width: 900px;
  display: flex;
  margin: 0 auto;
  justify-content: space-between;
  margin-bottom: 25px;
}

.button {
  margin-right: 20px;
  height: 90px;
  width: 150px;
  font-size: 25px;
  text-align: center;
  border-radius: 25px;


}