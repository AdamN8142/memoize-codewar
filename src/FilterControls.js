import React, { Component } from 'react'

class FilterControls extends Component {
  constructor() {
    super();
  }
  render() {
    return (
      <div>
        

      </div>
    )
  }
}


export default FilterControls;