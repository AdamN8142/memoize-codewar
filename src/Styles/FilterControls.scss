@import url('https://fonts.googleapis.com/css?family=Molle:400i');

.controls-container {
  width: 900px;
  display: flex;
  margin: 0 auto;
  justify-content: space-between;
  margin-bottom: 75px;

}

.button {
  height: 115px;
  width: 175px;
  font-size: 28px;
  text-align: center;
  border-radius: 25px;
  background-image: linear-gradient( red, orange,rgb(109, 109, 247), pink);
  color: white;
  font-weight: 800;
  font-family: 'Molle', cursive;
}

.button:hover {
  background-image: linear-gradient(purple, red, pink);
  color: black;
}

@media screen and (max-width:978px){

  .controls-container{
    display: flex;
    flex-direction: column;
    align-items: center;
    width: 50%
    }
    
    .button {
      width: 100%;
      height: 40px;
      margin-bottom: 30px;
    }
  
  }