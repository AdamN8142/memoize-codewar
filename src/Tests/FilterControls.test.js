import React from 'react';
import FilterControls from '../FilterControls';
import { shallow } from 'enzyme';


describe('FilterControls', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow (
      <FilterControls 
      data={this.state.data} 
      toggleComplete={this.toggleComplete} 
      completed={this.state.completed}
      />
    )
   })


})