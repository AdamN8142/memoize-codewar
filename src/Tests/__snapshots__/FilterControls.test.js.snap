// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterControls should match the snapshot will all data passed in 1`] = `
<div>
  <div
    className="controls-container"
  >
    <button
      className="button"
      id="lvl-1-btn"
      onClick={[Function]}
      value="8 kyu"
    >
      Level 1
    </button>
    <button
      className="button"
      id="lvl-2-btn"
      onClick={[Function]}
      value="7 kyu"
    >
      Level 2
    </button>
    <button
      className="button"
      id="lvl-3-btn"
      onClick={[Function]}
      value="6 kyu"
    >
      Level 3
    </button>
    <button
      className="button"
      id="completed-btn"
      onClick={[Function]}
      value="completed"
    >
      Completed
    </button>
  </div>
  <CardContainer
    data={
      Array [
        Object {
          "addedToStorage": false,
          "id": 1,
          "level": "8 kyu",
          "name": "Abbreviate a Two Word Name",
          "prompt": "Write a function to convert a name into initials. This kata strictly takes two words with one space in between them.",
          "replLink": "https://repl.it/@codesForDays/Abbreviate-a-Two-Word-Name?language=nodejs&folderId=",
          "type": "String",
        },
        Object {
          "addedToStorage": false,
          "id": 2,
          "level": "8 kyu",
          "name": "Welcome to the City",
          "prompt": "Create a method sayHello/say_hello/SayHello that takes as input a name, city, and state to welcome a person. Note that name will be an array consisting of one or more values that should be joined together with one space between each, and the length of the name array in test cases will vary.",
          "replLink": "https://repl.it/@codesForDays/Welcome-to-the-City?language=nodejs&folderId=",
          "type": "String",
        },
        Object {
          "addedToStorage": false,
          "id": 3,
          "level": "7 kyu",
          "name": "Remove Anchor from URL",
          "prompt": "Complete the function/method so that it returns the url with anything after the anchor (#) removed.",
          "replLink": "https://repl.it/@codesForDays/Remove-anchor-from-URL?language=nodejs&folderId=",
          "type": "String",
        },
        Object {
          "addedToStorage": false,
          "id": 4,
          "level": "7 kyu",
          "name": "Alphabet War",
          "prompt": "Write a function that accepts fight string consists of only small letters and return who wins the fight. When the left side wins return Left side wins!, when the right side wins return Right side wins!, in other case return Let's fight again!.",
          "replLink": "https://repl.it/@codesForDays/Alphabet-war?language=nodejs&folderId=",
          "type": "Fundamentals",
        },
      ]
    }
    filteredCategories={Array []}
    toggleComplete={[MockFunction]}
  />
</div>
`;
